charts/simple-workflow/templates/kmssecret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if and (.Values.secrets) (eq .Values.secretsEngine "kms") }}
```

Comment:
It seems we are using this same template across several apps (simple app, stateful app, simple workflow) Could we move it to nd-common

Reply:
Agreed. We can have another PR to extract this out from the existing charts and put it to `nd-common`.